components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.common/src/main/java/org/wso2/carbon/identity/api/server/configs/common/Constants.java

```diff
@@ -50,6 +50,8 @@ private Constants() {
 
     public static final String AUDIT = "AUDIT";
     public static final String CARBON = "CARBON";
+    public static final String AUDIT_LOWER_CASE = "audit";
```

Comment:
Let's refactor existing audit and carbon variables to lowercase, we don't need another 2 for lowercase.

Reply:
No need to change these constants since we are making the logType to uppercase in every scenario.